build.sc

```diff
@@ -15,8 +18,9 @@ val scalaNativeVersions = for {
   scalaNativeV <- Seq("0.4.0")
 } yield (scalaV, scalaNativeV)
 
-trait MainArgsPublishModule extends PublishModule with CrossScalaModule {
-  def publishVersion = "0.2.1"
+trait MainArgsPublishModule extends PublishModule with CrossScalaModule with Mima {
+  def publishVersion = VcsVersion.vcsState().format()
+  def mimaPreviousVersions = VcsVersion.vcsState().lastTag.toSeq
```

Comment:
This would silently accept a missing last tag, whereas we always want to use the last tag.